### PR TITLE
Top bar font and spacing.

### PR DESCRIPTION
**Pull request by Torgian**

### CODE REVIEW

**Comment by martinheidegger** [comment]
Lets turn this PR into a fix for all of the problems or, alternatively create new "sub-issues" for which this is a fix.

File: assets/main/sass/layout/_menu.scss [hunk 1]
```diff
@@ -136,7 +136,7 @@
         border-radius: 11px;
         max-height: 60px;
         height: 100%;
-        font-size: 24px;
+        font-size: 23px !important;
```
Comment (martinheidegger) [inline assets/main/sass/layout/_menu.scss]: Why does it need to be `!important` - Is there another selector overriding this?

Comment (Torgian) [reply assets/main/sass/layout/_menu.scss]: There was a styling in #menu that did over-ride it. I removed it in commit 5d9aeae.

**Comment by Torgian** [comment]
Referring #32 and #22 

I am adding a few screenshots here of mobile, tablet, and other views for the changes I have made so far in the top navigation bar. These include spacing / centering issues. Please review.

Tablet mode ( the second screenshot ) can be resolved a couple of different ways. I can either change the font-sizes and padding to make the links closer together, or change the menu so that the hamburger menu appears on tablet screens. I think the former would be better.

Screen Shot 2020-02-18 at 11 00 48 AM
Screen Shot 2020-02-18 at 11 00 39 AM
Screen Shot 2020-02-18 at 10 59 50 AM

**Comment by Torgian** [comment]
See this [commit](https://github.com/consento-org/consento-website/pull/61/commits/4efef7897358f36939710ffb0a9ffba38aee1071) for the following:

Before: 

Screen Shot 2020-02-18 at 11 00 39 AM

After fix:

Screen Shot 2020-02-18 at 11 00 48 AM


This is for tablet view. I reduced the font size from 1em to 0.5 em. This makes it look better and more fluid between different sizes. @marcchataigner @Tannybunny what do you think?

**Comment by marcchataigner-zz** [comment]
It looks better, more compact indeed. Maybe the visual could even be a bit compacted, so the title + text + visual would all fit on one screen?

**Comment by martinheidegger** [comment]
> Maybe the visual could even be a bit compacted, so the title + text + visual would all fit on one screen?

This PR is in regards to the menu spacing, not the visual, the menu spacing looks good to me. @Torgian maybe its a good idea to focus the screenshots on the relevant elements only?

**Comment by martinheidegger** [comment]
A rebase is needed to merge this PR.

**Comment by Torgian** [comment]
Rebase complete. This PR should resolve #22 and #32 and also contribute towards #13

**Comment by martinheidegger** [comment]
Merging for now. If a new problem arises, let's create a new issue.

**Pull request merged**